### DISCUSSION TITLE
Does encore.ts work in cloudflare workers?

### DISCUSSION THREAD

**Discussion by alexkreidler** [encoredev/encore#1533] [General]
The [benchmarking blog post](https://encore.dev/blog/event-loops) compares it against express and Hono (which is specifically designed for edge function environments). Wondering if the Rust/native module aspect of encore.ts allows it to deploy on edge function platforms like Workers, Lambda@Edge etc.

**Comment by pjborowiecki** [comment]
Anyone? :)

**Comment by marcuskohlberg** [comment]
You can’t currently deploy Encore onto Cloudflare Workers as Encore is designed more for use cases where you want to interact with actual infrastructure, like SQL databases, caching, pub/sub, etc. And in this case, running your compute closer to the database inside an actual cloud provider region is generally much more performant.

However, Encore + Cloudflare still work great together, by having Cloudflare running in front to give you the Auto-CDN capabilities.
E.g. Encore Cloud automatically sets up Cloudflare in front of your application.

**Reply by joshua1** [reply]
How do we use encore.dev with a cloudflare service like durable objects , or workflow service ? or is this not doable ?

**Comment by vivekrp** [comment]
@marcuskohlberg  Any progress towards this?
